src/cloud/docker/docker-config.md

```diff
@@ -46,6 +46,33 @@ To skip the interactive mode, use the `-n, --no-interaction` option.
 {:.bs-callout-info}
 The mode option for the `ece-docker build:compose` command does not affect the Magento mode. It determines the {{site.data.var.ece}} file system installation and read-only or read-write behavior.
 
+## Set the environment variables
+
+You can launch a Cloud Docker environment with predefined environment variables by adding the `--env-vars` option to the `ece-docker build:compose` command.
+
+For example,
+
+```bash
+bin/ece-docker build:compose --env-vars="{\"MAGENTO_CLOUD_VARIABLES\":{\"LOCK_PROVIDER\":\"db\",\"CRON_CONSUMERS_RUNNER\":{\"cron_run\":\"true\",\"max_messages\":5000,\"consumers\":[\"test\"]}}}"
```

Comment:
This appears all on one line, which is not too horrible if you want to cut and paste.
But if you want to see what is going on, then it's more difficult.

Reply:
I beefed up the intro sentence to show what's happening. Submitting json arrays in a command string is never pretty. ;-)